src/platform/packages/shared/kbn-monaco/src/console/lexer_rules/shared.ts

```diff
@@ -109,11 +110,14 @@ xjsonRules.json_root = [
   buildSqlStartRule(),
   // @ts-expect-error include a rule to start painless highlighting
   buildPainlessStartRule(),
+  // @ts-expect-error include a rule to start esql highlighting
```

Comment:
Nit: I think these comments should explain why there is a TS error expected

Reply:
I had auto-merge turned on, I'll get back to it again soon and will address this then